app/controllers/process_plates_controller.rb

```diff
@@ -36,9 +34,6 @@ def create
     # 20 barcodes takes about 2 mins as of 2020-06-22. This limit is to prevent the request timing out.
     raise "Please scan #{ProcessPlate::RECEIVE_PLATES_MAX} barcodes or fewer" if barcodes.count > ProcessPlate::RECEIVE_PLATES_MAX
 
-    responses = call_external_services(barcodes)
-    @results = generate_results(barcodes, responses)
-
```

Comment:
You're no longer setting @results, but are rendering the results template, which contains:

```haml
# app/views/process_plates/results.html.haml
  %tbody
    - @results.each do |barcode, details|
```

Not sure why this isn't blowing up, so sorry if I missed something. Still, probably makes sense to remove the results render, as the table is always going to be empty.

It used to just:
```
flash[:notice] = 'success'
...
format.html { redirect_to(new_process_plate_path) }
```

Although the new flash (which should be flash.now for a render) definitely looks like an improvement.